models/student.go

```diff
@@ -0,0 +1,34 @@
+package models
+
+import (
+	"github.com/fatih/structs"
+)
+
+type Student struct {
+	ID      string  `json:"id" structs:"id" bson:"_id" db:"id"`
+	Name    string  `json:"name" structs:"name" bson:"name" db:"name"`
+	Age     string  `json:"age" structs:"age" bson:"age" db:"age"`
+	GPA     float64 `json:"gpa" structs:"gpa" bson:"gpa" db:"gpa"`
+	ClassID string  `json:"classID" structs:"classID" bson:"classID" db:"classID"`
```

Comment:
json:"classID" structs:"classID" bson:"classID" db:"classID
is this the correct json ?

Reply:
i think it is . it shows no error in test